pkg/ccl/backupccl/show.go

```diff
@@ -863,14 +868,57 @@ func backupShowerFileSetup(inCol tree.Expr) backupShower {
 	}
 }
 
+// getManifestDirs gathers the path to the directory for each backup manifest,
+// relative to the collection root. In other words, path.Join(dest,
+// manifestDirs[i]) is the resolved manifest path. If the user passed
+// incremental_location, the path.Join(explicitIncPath,manifestDirs[i]) is the
```

Comment:
The comment references non-parameter variables making it a little hard to understand what the method is doing. Do you think we can rewrite it to be more self explanatory instead of having to read the code to understand the comment? The use of an example might help illustrate what this metho does.

Reply:
Good point. Done.